### DISCUSSION TITLE
Navigate to page with parameter

### DISCUSSION THREAD

**Discussion by InterfilEilev** [lepoco/wpfui#414] [Q&A] [answered]
How can i navigate to a view(Transient) and pass a parameter to it?

**Comment by giuseppetrematerra** [comment, marked as answer]
You could edit the IWindowService and make it call a method (called NavigatedFrom or NavigatedTo) implemented from an interface like the INavigationAware.

**Comment by lucazoffoli** [comment]
I've the same problem, how I can pass parameter with INavigationService?
